up.go

```diff
@@ -2,64 +2,53 @@ package goose
 
 import (
 	"database/sql"
+	"path/filepath"
 )
 
-// UpTo migrates up to a specific version.
-func UpTo(db *sql.DB, dir string, version int64) error {
-	migrations, err := CollectMigrations(dir, minVersion, version)
+// Up performs all types of migrations upwards, depending on the params.
+func Up(db *sql.DB, dir string, includeMissing bool, onlyOne bool, endVersion *int64) error {
```

Comment:
btw: There are API breaking changes; we would have to consider this for goose v3, but not earlier.